pkgs/dart_pad/lib/src/protos/dart_services.pb.dart

```diff
@@ -81,8 +95,21 @@ class CompileRequest extends $pb.GeneratedMessage {
   void clearReturnSourceMap() => clearField(2);
 }
 
+/// / Compile request for a multiple file set.
```

Comment:
This looks a bit awkward. Perhaps proto doc comments are meant to just use `//`? But maybe the generator should account for it either way.